internal/fs/vss_windows.go

```diff
@@ -810,6 +828,24 @@ func initializeVssCOMInterface() (*ole.IUnknown, error) {
 		}
 	}
 
+	// initialize COM security for VSS, this can't be called more then once
+	var capabilities uint32
+	if isPreWin8 {
+		capabilities = 0x00 // EOAC_NONE
+	} else {
+		capabilities = 0x20 // EOAC_STATIC_CLOAKING
+	}
```

Comment:
I think we can remove the pre win8 case entirely because [restic doesn't support it since v0.16.2](https://github.com/restic/restic/issues/4636)?

If we want to support it, we have to change more parameters than that. (impersonation level too)

Your thoughts @MichaelEischer ?